react/src/Analyses/index.tsx

```diff
@@ -355,6 +407,8 @@ export default function Analyses() {
                         exportAllData: true,
                         exportButton: { csv: true, pdf: false },
                         exportCsv: (columns, data) => exportCSV(columns, data, "Analyses"),
+                        rowStyle: data =>
+                            getHighlightColor(theme, data.priority, data.analysis_state),
```

Comment:
I'm not exactly sure why but this is only working for the first row.

Reply:
Weird... it will only highlight the first row that has a priority flag?